active/tab/tabcontrol.md

```diff
@@ -0,0 +1,252 @@
+section.  For example, see the Remarks for the MediaPlayerElement 
+(https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaPlayerElement#remarks). -->
+
+## Keyboarding behavior
+### TAB and Arrow key behavior
+The TabControl has areas containing left content, right content, and the Tab content (containing 0 or more Tabs). When the user hits the TAB key, focus moves between the content areas. When focus moves into the Tab content area, the selected Tab gains focus. Once the user is in the Tab content area, pressing the arrow key will move focus between the tabs. Arrow focus is trapped inside the Tab content area.
+
+### Selecting a Tab
+When a Tab has focus, pressing SPACE or ENTER will select that Tab.
+
+### Shortcuts for moving between tabs
+Ctrl+TAB will select the next Tab.
+Ctrl+Shift+TAB will select the previous Tab.
+Tab selection cycles (meaning if the user has selected the last tab and presses Ctrl+TAB, the first tab becomes selected).
+
+Ctrl+1 through 8 will automatically select the corresponding Tab.
+Ctrl+9 will automatically select the last Tab.
+
+### Closing a Tab 
+Hitting CTRL+W or CTRL+F4 will automatically close the selected Tab (assuming it is closable). 
+
+### Keyboard guidance for App Developers
+The above sections outline built-in keyboarding behavior provided by the TabControl. However, there are certain expected keyboard shortcuts that you will be responsible for implementing. 
+* If your Tab control supports adding a new Tab, users expect CTRL+T to open a new tab
+* Consider maintaining a list of recently closed Tabs. If the user presses CTRL+SHIFT+T, they expect recently closed tabs to be reopened.
+* If users can perform more commands on a Tab than just closing the Tab (for example, pinning a Tab or duplicating a Tab), consider adding a context menu to the Tab. 
+
+# API Notes
+<!-- Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. -->
+
+### TabControl properties and events
+
+| Property | Type | Description |
+|:-------- |:---- |:----------- |
+| CanCloseTabs | bool | Default value for the item if it doesn't specify a IsClosable value. |
+| CloseButtonOverlay | enum | Describes the behavior of the close button of unselected tabs. Values are {Auto, OnHover, Persistent}. Default is Auto. In WinUI 2.2, Auto maps to Persistent.  |
+| ItemHeaderTemplate | DataTemplate | Default template for the item if no template specified. |
+| SelectedTabWidth | double | The size of the selected tab header. |
+| TabHeader | object | Content to the left of the tab strip. |
+| TabHeaderTemplate | DataTemplate | Template for the Header. |
+| TabFooter | object | Content to the far right of the tab strip. |
+| TabFooterTemplate | DataTemplate | Template for the Footer. |
+| TabActionContent | object | Content immediately to the right of the tabs |
```

Comment:
I don't quite see the reason for this -- if we give them the Footer area, they can align items inside of there however they want.

Reply:
Rather than Header and Footer, perhaps it should be **Before** and **After** which will be contextually relevant whether Tabs are at the Top, Bottom, Left or Right - and would also work for localised situations.

Reply:
> Rather than Header and Footer, perhaps it should be Before and After which will be contextually relevant whether Tabs are at the Top, Bottom, Left or Right - and would also work for localised situations.

Interesting idea! Although we have precedent for generic areas before/after above/below left/right of a control, we don't have precedent for the wording "Before" and "After". Instead, we traditionally use the words "Header" and "Footer". 

Given that context, we are leaning towards having a single content area to the right of the Tab strip area called the "TabStripFooter".